app/models/account_request.rb

```diff
@@ -23,6 +23,10 @@ class AccountRequest < ApplicationRecord
 
   has_one :organization, dependent: :nullify
 
+  enum status: %w[requested pending approved rejected].map { |v| [v, v] }.to_h
```

Comment:
```suggestion
  enum status: %w[started pending_confirmation pending_approval approved rejected].map { |v| [v, v] }.to_h
```
What do you think about this?

Reply:
Hm... I think using `pending` should only make sense if there's only one possible thing that's pending. `pending_approval` doesn't indicate that it's been confirmed, for example. I actually also don't like the fact that `confirm` and `approve` are pretty much synonyms of each other. How about this?
* started
* user_confirmed
* admin_approved
* rejected

Reply:
I liked that one much more! Sounds good.